src/main/java/frc/robot/subsystems/SwerveModule.java

```diff
@@ -33,14 +30,12 @@ public class SwerveModule {
     private AnalogEncoder _rotEncoder;
     private RelativeEncoder _transEncoder;
 
-    public SwerveModule(int rotID, int transID, int rotEncoderID, int transEncoderID,
-            double rotEncoderOffset, boolean rotInverse, boolean transInverse, PIDController rotPID,
-            PIDController transPID) {
+    public SwerveModule(int rotID, int transID, int rotEncoderID, double rotEncoderOffset,
```

Comment:
Too many parameters. Abstract them into SwerveModuleConfig object that can be loaded from a json file